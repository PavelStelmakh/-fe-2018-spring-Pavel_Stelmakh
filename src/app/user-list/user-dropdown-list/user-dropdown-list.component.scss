:host {
    position: absolute;
    display: block;
    top: 142px;
    width: 500px;
    background-color: rgba(0, 0, 0, 0.7);
    box-shadow: 0 0 10px 3px rgba(0,0,0,0.5);
}