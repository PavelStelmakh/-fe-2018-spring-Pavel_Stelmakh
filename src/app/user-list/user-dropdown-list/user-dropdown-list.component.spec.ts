import { TestBed, async, ComponentFixture } from '@angular/core/testing';
import { NO_ERRORS_SCHEMA } from '@angular/core';
import { UserDropdownListComponent } from './user-dropdown-list.component';

describe('UserDropdownListComponent', () => {
  let fixture: ComponentFixture<UserDropdownListComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      declarations: [
        UserDropdownListComponent
      ],
      schemas: [
       NO_ERRORS_SCHEMA
      ]
    }).compileComponents();

  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(UserDropdownListComponent);
  });

  it('renders markup to snapshot', () => {
    expect(fixture).toMatchSnapshot();
  });

});
