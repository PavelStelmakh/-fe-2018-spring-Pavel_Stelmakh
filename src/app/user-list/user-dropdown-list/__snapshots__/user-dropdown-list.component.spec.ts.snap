// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserDropdownListComponent renders markup to snapshot 1`] = `
<app-user-dropdown-list>
  
</app-user-dropdown-list>
`;
